src/Components/SignupForm/SignupForm.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import { useFirebase } from "react-redux-firebase";
+
+import SignupFormView from "./SignupFormView";
+
+const SignupForm = function () {
+  const firebase = useFirebase();
+  const [error, setError] = useState(null);
+
+  const handleSignup = (email, password, passwordConfirm) => {
```

Comment:
Maybe we can add some check to see if the email is valid? Or is this handled ny firebase itself

Reply:
Yep, it's handled by firebase already :)